cmd/system-probe/subcommands/run/command.go

```diff
@@ -193,6 +197,13 @@ func startSystemProbe(cliParams *cliParams, log log.Component, sysprobeconfig sy
 
 	setupInternalProfiling(sysprobeconfig, configPrefix, log)
 
+	if ddconfig.Datadog.GetBool("remote_configuration.enabled") {
```

Comment:
The only configuration currently supported is `log_level` (see https://github.com/DataDog/datadog-agent/blob/f939d0ae8c32ea27598f26f454b8d8903b12314d/comp/remote-config/rcclient/rcclient.go#L94-L132)

Reply:
I meant the functionality to make RC work, not what transits over it. I think your other answer about the architecture answers this question though.

Reply:
I think my question still applies, so I'm going to try and clarify. Within system-probe, we only read `datadog.yaml` without any of the secrets replaced. This is because `system-probe` runs as `root`, and thus cannot execute the same secrets backend the `agent` does.

If any of the configuration values RC needs to operate (not the configuration values that can be changed at runtime) could be obscured by secrets, then this setup will not work in `system-probe`.